# 
# policy_path: "/home/csl/test/legged_rl_ws/src/legged_rl/legged_gym/logs/rough_Reddog/exported/policies/policy_1.pt"
policy_path: "/home/csl/rdog/reddog_ros2_ws/src/redDog_RL_Him/controller/rl_quadruped_controller/rl_quadruped_controller/pre_train/policy_him2.pt"
xml_path: "scene_terrain_1.xml"

# Total simulation time
simulation_duration: 60.0
# Simulation time step
simulation_dt: 0.005
# Controller update frequency (meets the requirement of simulation_dt * controll_decimation=0.02; 50Hz)
control_decimation: 4

# kps: [3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3]
# kds: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
kps: [ 10,  10,  8,  8,  8,  8,  8,  8,  8,  8,  8,  8]
kds: [0.4, 0.4, 0.4, 0.4, 0.4, 0.4, 0.4, 0.4, 0.4, 0.4, 0.4, 0.4]


default_angles: [0.1, 0.785, -1.57, -0.1, 0.785, -1.57, 0.1, -0.785, 1.57, -0.1, -0.785, 1.57]

lin_vel_scale: 1.0
ang_vel_scale: 1.0
dof_pos_scale: 1.0
dof_vel_scale: 0.05
action_scale: 0.2 # 0.3
cmd_scale: [2.0, 2.0, 0.25]
num_actions: 12
num_obs: 270
one_step_obs_size: 45
obs_buffer_size: 6

cmd_init: [ 0.0 , 0.0, 0.0]